emukit/core/loop/stopping_conditions.py

```diff
@@ -22,7 +27,8 @@ class FixedIterationsStoppingCondition(StoppingCondition):
     """ Stops after a fixed number of iterations """
     def __init__(self, i_max: int) -> None:
         """
-        :param i_max: Maximum number of function observations within the loop, excluding initial points
+        :param i_max: Maximum number of function
+        observations within the loop, excluding initial points
```

Comment:
```suggestion
                observations within the loop, excluding initial points
```

normally you want some additional indent for multiline param docstrings like that